streampark-common/src/main/scala/org/apache/streampark/common/util/YarnUtils.scala

```diff
@@ -127,8 +128,8 @@ object YarnUtils extends Logger {
         val conf = HadoopUtils.hadoopConf
         val useHttps = YarnConfiguration.useHttps(conf)
         val (addressPrefix, defaultPort, protocol) = useHttps match {
-          case x if x => (YarnConfiguration.RM_WEBAPP_HTTPS_ADDRESS, "8090", "https://")
-          case _ => (YarnConfiguration.RM_WEBAPP_ADDRESS, "8088", "http://")
+          case x if x => (YarnConfiguration.RM_WEBAPP_HTTPS_ADDRESS, "8090", Constant.HTTPS_SCHEMA)
+          case _ => (YarnConfiguration.RM_WEBAPP_ADDRESS, "8088", Constant.HTTP_SCHEMA)
```

Comment:
Hi, @RocMarshal  it is recommended that the port number here be extracted as a constant.

Reply:
Hi, @wuchunfu Thanks for the review .

<img width="926" alt="image" src="https://github.com/apache/incubator-streampark/assets/64569824/9d5c8b5b-a1a6-4987-9593-d8ae02699ccd">

I rechecked the rule. 
I'd appreciated with your confirmation~

Reply:
@RocMarshal Thank you for reminding me. I remembered it wrong. I agree with you.